app/components/BuyRamForm/buyRamFormStyle.js

```diff
@@ -0,0 +1,11 @@
+const buyRamFormStyle = {
```

Comment:
For now these radio styles are within `BuyRamForm`, but we can move them in `regularFormStyle` in the future if needed across more forms. However I want to stick to `no early optimizations` approach for now